service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderControllerAboutToStartTest.java

```diff
@@ -101,6 +136,34 @@ void shouldPopulateJudgeAndLegalAdvisorInUploadRoute() {
         assertThat(responseCaseData.getJudgeAndLegalAdvisor()).isEqualTo(judgeAndLegalAdvisor);
     }
 
+    @Test
+    void shouldPopulateSDODirectionsWhenDirectionsAreEmpty() {
```

Comment:
tbh, as this is integration test, i would not mock stadnarddirectionService, i would write tests like this;

    @Test
    void shouldPopulateSDODirectionsWhenDirectionsAreEmpty() {
        CaseData originalCaseData = CaseData.builder().build();

        CaseData actualCaseData = extractCaseData(postAboutToStartEvent(originalCaseData));

        assertThat(actualCaseData.getAllParties()).hasSize(5);
        assertThat(actualCaseData.getLocalAuthorityDirections()).hasSize(7);
        assertThat(actualCaseData.getRespondentDirections()).hasSize(1);
        assertThat(actualCaseData.getOtherPartiesDirections()).hasSize(1);
        assertThat(actualCaseData.getCafcassDirections()).hasSize(3);
        assertThat(actualCaseData.getCourtDirections()).hasSize(1);
    }

    @Test
    void shouldNotOverwriteSDODirectionsWhenDirectionsAreNotEmpty() {
        CaseData originalCaseData = CaseData.builder()
            .localAuthorityDirections(wrapElements(Direction.builder().assignee(LOCAL_AUTHORITY).build()))
            .build();

        CaseData actualCaseData = extractCaseData(postAboutToStartEvent(originalCaseData));

        assertThat(actualCaseData.getAllParties()).isEqualTo(originalCaseData.getAllParties());
        assertThat(actualCaseData.getLocalAuthorityDirections()).isEqualTo(originalCaseData.getLocalAuthorityDirections());
        assertThat(actualCaseData.getRespondentDirections()).isEqualTo(originalCaseData.getRespondentDirections());
        assertThat(actualCaseData.getOtherPartiesDirections()).isEqualTo(originalCaseData.getOtherPartiesDirections());
        assertThat(actualCaseData.getCafcassDirections()).isEqualTo(originalCaseData.getCafcassDirections());
        assertThat(actualCaseData.getCourtDirections()).isEqualTo(originalCaseData.getCourtDirections());
    }